### ISSUE TITLE
construct a template for Shiny in mosaic

### ISSUE THREAD

**Issue by nicholasjhorton** [ProjectMOSAIC/mosaic#491]
It would be lovely to have a new template in the mosaic package that built the same Shiny app as the default "Shiny in markdown" but using lattice rather than base graphics to construct the histogram.

A second template doing the same thing using ggplot2 would be cool to consider.

**Comment by rpruim** [comment]
I've added a shiny template using lattice graphics.  Now that I've done it, I'm wondering about a couple things:

 1) Rather than making multiple versions, I'm thinking we should let the user toggle between lattice and ggplot2 inside the document.

 2) I've recreated the document from the shiny package, but we might prefer to do something more interesting and more directly connected to the mosaic package here.  Some possibilities:

 * demonstrate `do()` and resampling
 * illustrate extra features of `histogram()` with the mosaic package (ie. `width`, `center`, `fit`)
 * illustrate `freqpolygon()` and or `ashplot()`
 * illustrate `plotDist()`

Any other ideas?  I'm leaning toward `freqpolygon()` and `ashplot()`
